src/plays/airbnb-experiences-clone/Readme.md

```diff
@@ -0,0 +1,27 @@
+# AirBnb Experiences Clone
+
+This project is the clone of the AirBnb Experiences page.
+
+## Play Demographic
+
+- Language: js
+- Level: Beginner
+
+## Creator Information
+
+- User: aaqib605
+- Gihub Link: https://github.com/aaqib605
+- Blog: https://aaqibjavaid.hashnode.dev/
+- Video: 
+
+## Implementation Details
+
+Update your implementation idea and details here

```

Comment:
Can we add implementation details here? so that others can get an idea of how this play has been built.

Reply:
Sure, will do.

Reply:
I have added the implementation details. Committed changes. Now upon trying to push the changes, I get the following error.
<img width="1440" alt="Screenshot 2023-11-02 at 12 31 13 PM" src="https://github.com/reactplay/react-play/assets/78491606/22b16da5-807c-4047-949a-18d22b25a3fa">

Reply:
Before pushing take the pull and then push the changes

Reply:
@Sachin-chaurasiya Done, thanks.